testData/solutions/win32.json5

```diff
@@ -3708,6 +3708,322 @@
+                                "Tile": "0",
+                                "Center": "0",
+                                "Span": "22"
+                            },
+                            "noMatch": {
+                                "outputValue": "6"
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.desktopBackground.Scaling": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "WallpaperStyle",
+                            "match": {
+                                "6": "Fill",
+                                "10": "Fit",
+                                "2": "Stretch",
+                                "0": {
+                                    "outputValue": {
+                                        "transform": {
+                                            "type": "fluid.transforms.valueMapper",
+                                            "defaultInputPath": "TileWallpaper",
+                                            "match": {
+                                                "1": "Tile",
+                                                "0": "Center"
+                                            },
+                                            "noMatch": {
+                                                "outputValue": "Tile"
+                                            }
+                                        }
+                                    }
+                                },
+                                "22": "Span"
+                            },
+                            "noMatch": {
+                                "outputValue": "Fill"
+                            }
+                        }
+                    }
+                }
+            },
+            "configureImage": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETDESKWALLPAPER",
+                    "setAction": "SPI_SETDESKWALLPAPER",
+                    "uiParam": 260,
+                    "pvParam": {
+                        "type": "array",
+                        "valueType": "TCHAR",
+                        "length": 260
+                    }
+                },
+                "supportedSettings": {
+                    "ImageConfig": {
+                        "schema": {
+                            "title": "Desktop background wallpaper",
+                            "description": "The path to the image to be set as the new desktop wallpaper.",
+                            "type": "string",
+                            "default": "C:\\Windows\\Web\\Wallpaper\\Windows\\img0.jpg"
```

Comment:
This default should use one of the standard environment variables to locate the Windows installation, e.g. %SYSTEMROOT

Reply:
Great, changing it.